hera_mc/cm_active.py

```diff
@@ -5,8 +5,44 @@
 
 """Methods to load all active data for a given date."""
 
-from . import cm_utils
+from . import mc, cm_utils
 from . import cm_partconnect as partconn
+from copy import copy
```

Comment:
super minor, but this line should be at the top of the imports with an empty line before the imports from the hera_mc package.